actionpack/test/dispatch/routing/inspector_test.rb

```diff
@@ -421,8 +421,6 @@ def test_regression_route_with_controller_regexp
       end
 
       def test_inspect_routes_shows_resources_route_when_assets_disabled
```

Comment:
This test is now the same as this other test https://github.com/rails/rails/blob/bf67e5de57dd9481d78494178ca9905e01bf4510/actionpack/test/dispatch/routing/inspector_test.rb#L69-L78

Are both needed? 

I'm not sure what value this test is adding, but I didn't want to tear it out right away.

Reply:
Can you remove the whole test instead? The `@set` might have been necessary when the test was added 8 years ago, but probably isn't anymore.